src/Microsoft.AspNetCore.Mvc.Core/Internal/IControllerPropertyActivatorFactory.cs

```diff
@@ -1,10 +1,13 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using Microsoft.AspNetCore.Mvc.Controllers;
+
 namespace Microsoft.AspNetCore.Mvc.Internal
 {
-    public interface IControllerPropertyActivator
+    public interface IControllerPropertyActivatorFactory
```

Comment:
`Provider` rather than factory would be consistent with the other changes in this PR